app/src/main/java/org/eyeseetea/malariacare/SettingsActivity.java

```diff
@@ -260,11 +440,11 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
     @TargetApi(Build.VERSION_CODES.HONEYCOMB)
     public static class GeneralPreferenceFragment extends PreferenceFragment {
 
-        private AutoCompleteEditTextPreference autoCompleteEditTextPreference;
         @Override
         public void onCreate(Bundle savedInstanceState) {
             super.onCreate(savedInstanceState);
             addPreferencesFromResource(R.xml.pref_general);
+            addPreferencesFromResource(R.xml.pref_general);
```

Comment:
@adrianq this is just an awful copy&pase error, this code was running for just for tablets. Now it is fixed.